examples/pytorch/pytorch_lightning_simple.py

```diff
@@ -17,6 +17,7 @@
 
 from packaging import version
 import pytorch_lightning as pl
+from pytorch_lightning import Callback
```

Comment:
I think you revert the changes in #2294. Could you remove the changes in this file?